### ISSUE TITLE
bootstrap_form no_error_top

### ISSUE THREAD

**Issue by danniranderis** [dyve/django-bootstrap3#372]
Hi.

I have struggled with `{% bootstrap_form form %}` for a while now - it seems to me, that it's not possible to tell the templatetag _not_ to render errors in a message in the top (but still on each field) - or that it is possible to customize that message. Am I correct, or am I overseeing something?

Right now I have settled with looping over the fields as shown here. Do I have to stick with that?
```python
{% for field in form %}
    {% bootstrap_field field %}
{% endfor %}
```

Thanks in advance
Danni

**Comment by potar** [comment]
The quickest solution is to patch the option:
```
from bootstrap3 import renderers
renderers.FormRenderer.render_errors.__defaults__ = ('non_fields',)
```
The better one would be to add a proper option to Django settings instead of `all`:
https://github.com/dyve/django-bootstrap3/blob/develop/bootstrap3/renderers.py#L199

**Comment by dyve** [comment]
Fixed with the new error display options in 9.0.0
